### ISSUE TITLE
Enable laravel-blade-snippets to run on VS Code on the Web

### ISSUE THREAD

**Issue by isidorn** [onecentlin/laravel-blade-snippets-vscode#143]
:wave: I'm Isidor, a PM from the VS Code team, and we recently announced VS Code for the Web at https://vscode.dev/, which provides a free, zero-install Microsoft Visual Studio Code experience running entirely in your browser.  You can learn more [here](https://code.visualstudio.com/docs/editor/vscode-web). 

We'd like to ensure laravel-blade-snippets can run in the web, and we have a guide for enabling extensions for the web [here](https://code.visualstudio.com/api/extension-guides/web-extensions). As a couple of highlights:

- In VS Code for the Web, both the UI and extension host run inside the browser.
- A web extension is structured like a regular extension, but with a different main file: it's defined by the `browser` property
- Access to workspace files needs to go through the VS Code file system API accessible at `vscode.workspace.fs`
- There are currently [three ways](https://code.visualstudio.com/api/extension-guides/web-extensions#test-your-web-extension) to test a web extension

I saw that you have some node dependencies that might be super easy to replace, however I wanted to create this issue so we get the conversation going. 
Also I see that your extension is contributing `grammars`, so it might make sense to use `when` clauses and disable other contributions which do not work on the web, but make sure that `grammars` are still contributed

Please let us know if you have questions or feedback on enabling your extension for the web - our team would love to learn from your experiences and collaborate. Thank you!

**Comment by bamurtaugh** [comment]
Hi @onecentlin! I was wondering if you might have thoughts on this? We're happy to discuss if you have any questions or feedback. Thank you!

**Comment by onecentlin** [comment]
@bamurtaugh Yes, I would like to support this extension for the web. Since this extension contains language server, which was contributed by one of the contributors, I am not sure if it would be effected.

**Comment by bamurtaugh** [comment]
Thanks for your response @onecentlin!

> Since this extension contains language server, which was contributed by one of the contributors, I am not sure if it would be effected.

@tanhakabir thoughts?
